packages/repository-json-schema/src/__tests__/unit/build-schema.unit.ts

```diff
@@ -232,13 +232,18 @@ describe('build-schema', () => {
       expect(key).to.equal('modelPartial');
     });
 
+    it('returns "excluding[]" when a single option "exclude" is set', () => {
+      const key = buildModelCacheKey({exclude: []});
+      expect(key).to.equal('modelExcluding[]');
+    });
+
     it('returns concatenated option names otherwise', () => {
       const key = buildModelCacheKey({
-        // important: object keys are defined in reverse order
         partial: true,
+        exclude: [],
         includeRelations: true,
       });
-      expect(key).to.equal('modelPartialWithRelations');
+      expect(key).to.equal('modelPartialExcluding[]WithRelations');
```

Comment:
I find this ugly. Let's treat `exclude: []` as if it was not provided at all.

```suggestion
      expect(key).to.equal('modelPartialWithRelations');
```

I am proposing the following:
- Add a new test to verify `buildModelCacheKey({exclude: []})`
- Modify this test to add some items to `exclude` configuration